### DISCUSSION TITLE
Harbor 2.10 smtp config not working - emails not sending

### DISCUSSION THREAD

**Discussion by salahawad** [goharbor/harbor#19927] [Q&A] [answered]
Hello,

do you know where to set the smtp config i have added the below on the bottom of harbor.yml file but it is not working 
email:
  host: smtp.gmail.com
  port: 587
  username: myemail
  password: mypassword
  from: myemail
  ssl: false / i tried this to true too
  insecure: false

**Comment by zyyw** [comment, marked as answer]
@salahawad The email configuration feature is removed from Harbor since v2.7.0:
* https://github.com/goharbor/harbor/releases/tag/v2.7.0

FYI:
https://goharbor.io/docs/2.6.0/administration/general-settings/
v.s.
https://goharbor.io/docs/2.7.0/administration/general-settings/

**Reply by salahawad** [reply]
How can we rest password and configure the system for getting registry scan results and others

**Reply by zyyw** [reply]
FYI:
<img width="908" alt="Screenshot 2024-02-05 at 16 48 23" src="https://github.com/goharbor/harbor/assets/6709992/f0de5409-cdf9-4d9e-938b-24e34a4b45a2">

**Comment by inlann** [comment]
> @salahawad The email configuration feature is removed from Harbor since v2.7.0:
> 
> * https://github.com/goharbor/harbor/releases/tag/v2.7.0
> 
> FYI: https://goharbor.io/docs/2.6.0/administration/general-settings/ v.s. https://goharbor.io/docs/2.7.0/administration/general-settings/

Hello,

May I ask why harbor deprecates the smtp support? I only found two related PRs: #17712 and #17711
